client/verta/verta/_registry/modelversion.py

```diff
@@ -234,8 +234,11 @@ def log_model(self, model, custom_modules=None, model_api=None, artifacts=None,
         if artifacts:
             self.add_attribute(_MODEL_ARTIFACTS_ATTR_KEY, artifacts, overwrite=overwrite)
 
-        if isinstance(model, six.string_types):  # filepath
-            model = open(model, 'rb')
+        if isinstance(model, six.string_types):
```

Comment:
Now it looks to me like this handling of file or directory should happen internally to `_artifact_utils.serialize_model` because the same logic is getting copied in `modelversion.py` and in both `log_model` methods in `experimentrun.py`. Do you think that small refactor is small enough to do in the scope of this PR?